flake.nix

```diff
@@ -271,66 +271,59 @@
         });
```

Comment:
What are all these nix changes for?

Reply:
I think he accidentally pulled in some of @dpc's commits